CRM/CiviAwards/Helper/ApplicantReview.php

```diff
@@ -24,4 +24,22 @@ public static function getActivityTypeId() {
     return $result['values'][0]['value'];
   }
 
+  /**
+   * Return the Custom Group ID attached to the Applicant review activity type.
+   */
+  public static function getApplicantReviewCustomGroupId() {
+    $applicantReviewCustomGroup = [];
+    $result = civicrm_api3('CustomGroup', 'get', [
+      'sequential' => 1,
+      'extends' => 'Activity',
+      'extends_entity_column_value' => self::getActivityTypeId(),
+    ]);
+
+    if (!empty($result['values'])) {
```

Comment:
@shahrukh-compucorp Yes, you are right. I fixed this here https://github.com/compucorp/uk.co.compucorp.civiawards/pull/166/commits/1376634969814cfcf7ce50c3070dbe0deaea3543
I use `getsingle` because I only need one result, and having less than one would imply that something is wrong.